internal/statestore/backfill.go

```diff
@@ -200,18 +200,18 @@ func (rb *redisBackend) UpdateBackfill(ctx context.Context, backfill *pb.Backfil
 	return nil
 }
 
-// AcknowledgeBackfill stores Backfill's last acknowledgement time.
+// UpdateAcknowledgmentTimestamp stores Backfill's last acknowledgement time.
 // Check on Backfill existence should be performed on Frontend side
-func (rb *redisBackend) AcknowledgeBackfill(ctx context.Context, id string) error {
+func (rb *redisBackend) UpdateAcknowledgmentTimestamp(ctx context.Context, id string) error {
 	redisConn, err := rb.redisPool.GetContext(ctx)
 	if err != nil {
-		return status.Errorf(codes.Unavailable, "AcknowledgeBackfill, id: %s, failed to connect to redis: %v", id, err)
+		return status.Errorf(codes.Unavailable, "UpdateAcknowledgmentTimestamp, id: %s, failed to connect to redis: %v", id, err)
 	}
 	defer handleConnectionClose(&redisConn)
-	return acknowledgeBackfill(redisConn, id)
+	return updateAcknowledgmentTimestamp(redisConn, id)
 }
 
-func acknowledgeBackfill(conn redis.Conn, backfillID string) error {
+func updateAcknowledgmentTimestamp(conn redis.Conn, backfillID string) error {
```

Comment:
Maybe it is better to use prefix 'do' like in other place?

Reply:
Thanks for the hint will add `do`